string vehicle
string bag_time
string sensor_id
uint64 time_step

